src/main/groovy/de/tracetronic/jenkins/plugins/ecutestexecution/clients/RestApiClientV1.groovy

```diff
@@ -35,12 +35,20 @@ import java.util.concurrent.TimeoutException
 class RestApiClientV1 implements RestApiClient{
 
     private ApiClient apiClient
+    private boolean executionTimedOut = false
```

Comment:
you could make it public and just set the prop. The method name toggle is just wrong 😉